resources/public/api/conf/2.0/application.raml

```diff
@@ -151,6 +153,11 @@ traits:
           type: datetime
           description: This is an optional query parameter that you can use to filter for departure declarations updated after a specific date and time.
           example: 2021-06-21T09:00+00:00
+        movementEORI:
+          required: false
+          type: string
+          description: This is an optional query parameter that you can use to filter for departure declarations that were submitted with a specified movement EORI.
+          example: ezv3Z
```

Comment:
"[A-Z]{2}[!-~]{1,15}" Looks like first two letters should be capital letters

Reply:
Yep, best for this to be a representative example, so something like `GB1234567890`